src/Converter/Feature2Mesh.js

```diff
@@ -496,7 +557,17 @@ function featureToMesh(feature, options) {
     var mesh;
     switch (feature.type) {
         case FEATURE_TYPES.POINT:
-            mesh = featureToPoint(feature, options);
+            if (feature.style.point?.model?.object) {
+                try {
+                    mesh = pointsToInstancedMeshes(feature);
+                    mesh.isInstancedMesh = true;
+                } catch (e) {
+                    console.error(e);
```

Comment:
Just thought about something : do we need to log the error since the error message is thrown above ?